app/assets/stylesheets/sul-variables.scss

```diff
@@ -15,6 +16,7 @@ $sul-footer-bg-color: $beige-10-percent;
 $sul-h1-font-color: $gray;
 $sul-h2-border-color: $beige-30-percent;
 $sul-h2-font-color: $cardinal-red;
+$sul-form-input-focus: $mostly-pure-orange;
```

Comment:
It doesn't look like the `$sul-form-input-focus` is used anywhere (I guess `scss-lint` doesn't catch that).